lessons/r/dplyrmagrittr/lesson.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: dplyr and magrittr
+visible: true
+tags:
+  - r
+  - intermediate
+---
+
+ - **Authors**: Ahmed Hasan
+ - **Research field**: Genomics
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/r/dplyrmagrittr>
+
+## Topics Covered:
+
+- 'tidy' data
+- `dplyr` verbs:
+  - `select`
+  - `filter`
+  - `arrange`
+  - `mutate`
+  - `summarise`
+- `magrittr`
+  - the pipe (`%>%`)
+  - tee operator (`%T>%`)
+  - compound assignment (`%<>%`)
+
+### What You Will Need
+
+- R 3.3.1+ with `dplyr` and `magrittr` installed
+  - To install both packages, run `install.packages(c('dplyr', 'magrittr'))` in the command line
+
+If you'd like to follow along in Jupyter Notebook, do the following:
```

Comment:
Imagine how reduced the cognitive load would be if there was one IDE that was awesome AND multilingual so that there never was a need learning new ones!!?? Oh wait... that's the Jupyter notebook... 🔥 🔫 

However, I will say this, I much prefer reading an R-markdown source file over the raw ipynb-notebook format. It's interesting that R-markdown have started supporting additional languages now, I wonder if it will catch on for non-R users. Hopefully innovation from both camps will lead to great products for us in the end =)